pnp_bot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers --------------------------------------
  Rev22_position_controller:
    type: effort_controllers/JointPositionController
    joint: Rev22
    pid: {p: 0, i: 0, d: 0}
  Rev23_position_controller:
    type: effort_controllers/JointPositionController
    joint: Rev23
    pid: {p: 0, i: 0, d: 0}
  Rev24_position_controller:
    type: effort_controllers/JointPositionController
    joint: Rev24
    pid: {p: 0, i: 0, d: 0}
  Rev25_position_controller:
    type: effort_controllers/JointPositionController
    joint: Rev25
    pid: {p: 0, i: 0, d: 0}
  Rev27_position_controller:
    type: effort_controllers/JointPositionController
    joint: Rev27
    pid: {p: 5.0, i: 0.00005, d: 4.0}
  Rev29_position_controller:
    type: effort_controllers/JointPositionController
    joint: Rev29
    pid: {p: 0.8, i: 0.0001, d: 2.5}
